ui/pages/login/index.js

```diff
@@ -201,12 +218,14 @@ export default function Login() {
             data-testid='form-field-password'
             onChange={e => {
               setPassword(e.target.value)
+              setErrors({})
               setError('')
             }}
             className={`mt-1 block w-full rounded-md  shadow-sm focus:border-blue-500 focus:ring-blue-500 sm:text-sm ${
-              error ? 'border-red-500' : 'border-gray-300'
+              errors.password ? 'border-red-500' : 'border-gray-300'
```

Comment:
Is there a case where we return an error specific to the password on login? The server should be vague in its error in this case so we don't leak info.